libs/payments/eligibility/src/lib/eligibility.service.ts

```diff
@@ -72,6 +75,9 @@ export class EligibilityService {
     countryCode?: string
   ) {
     if (!countryCode) {
+      this.statsd.increment('eligibility_service_get_product_availability', {
```

Comment:
I'm not sure adding the statsd recording to the `getProductAvailabilityForLocation` method is the right approach since this method can be called in multiple places, and therefore won't give an accurate reading on how many times the `/location` page is reached. (I believe it's currently already being used on the `/new` page)

As an alternative approach, have you considered extending the Emittery events in the PaymentsEmitterService, which can then record the statsd metric, and emitting an event on the `/location` page, on page load?

Reply:
I like the emitter service approach, and added an event similar to the sp3 rollout one